src/kernel/sched/sched.c

```diff
@@ -416,6 +416,7 @@ static void sched_ticker_update(void) {
 	cur = schedee_get_current();
 
 	next = runq_get_next(&rq.queue);
+	if(next == NULL) goto empty_thread_queue;
```

Comment:
Could you write this part more carefuly?
I mean use 
```
if (next == NULL ) {

}
```
Execept this, it's great if you add a comment

Reply:
Ok, No Problem